src/js/components/tasks/capture/index.tsx

```diff
@@ -141,9 +191,39 @@ const CaptureModal = ({
     setInputs,
   });
 
+  useFormDefaults({
+    field: 'ignored_changes',
+    value: defaultIgnoredChanges,
+    inputs: ignoredInputs,
+    setInputs: setIgnoredInputs,
+  });
+
   const dirSelected = Boolean(inputs.target_directory);
-  const changesChecked = Object.values(inputs.changes).flat().length;
   const hasCommitMessage = Boolean(inputs.commit_message);
+  const changesChecked = Object.values(inputs.changes).flat().length || 0;
+  const ignoredChecked =
+    Object.values(ignoredInputs.ignored_changes).flat().length || 0;
+  const bothChecked = changesChecked && ignoredChecked;
+
+  const nextPage = () => {
+    setPageIndex(pageIndex + 1);
+  };
+
+  const saveIgnored = (e: React.FormEvent<HTMLFormElement>) => {
+    submitIgnored(e);
+    // dispatch(
+    //   updateObject({
+    //     objectType: OBJECT_TYPES.ORG,
+    //     url: window.api_urls.scratch_org_detail(orgId),
+    //     data: {
+    //       task: taskId,
+    //       org_type: ORG_TYPES.DEV,
+    //       ignored_changes: inputs.changes,
+    //     },
+    //     hasForm: true,
+    //   }),
+    // );
+  };
```

Comment:
RE: https://github.com/oddbird/Metecho/pull/261#issuecomment-618447186

OK, it makes sense that you'd see that; it could be an issue with how I am using `useForm`. Try un-commenting this call to dispatch instead of calling `submitIgnored()`; you should be able to reproduce it.